library_generation/cli/entry_point.py

```diff
@@ -113,6 +113,7 @@ def generate(
         # baseline_generation_config is not specified.
         # Do not generate pull request description.
         generate_from_yaml(
+            config_path=current_generation_config,
```

Comment:
Not part of this PR, `current_generation_config` and `baseline_generation_config` are actually paths not the content, so probably better to rename them as something like `current_generation_config_path`.

Reply:
Raised #2638.